src/org/usfirst/frc/team2706/robot/commands/teleop/GearHandlerOpen.java

```diff
@@ -0,0 +1,24 @@
+package org.usfirst.frc.team2706.robot.commands.teleop;
+
+import org.usfirst.frc.team2706.robot.Robot;
+
+import edu.wpi.first.wpilibj.DoubleSolenoid;
+import edu.wpi.first.wpilibj.Joystick;
+import edu.wpi.first.wpilibj.command.Command;
+
+public class GearHandlerOpen extends Command {
```

Comment:
Same as GearHandlerClosed, documentation and hopefully get rid of this class and merge it into GearHandler